tensorflow/python/ops/array_ops.py

```diff
@@ -1066,7 +1066,7 @@ def concat(values, axis, name="concat"):
                                   name=name)
 
 
-def boolean_mask(tensor, mask, name="boolean_mask"):
+def boolean_mask(tensor, mask, axis=None, name="boolean_mask"):
```

Comment:
For backwards compatibility, all new arguments must be last (in addition to having defaults).